### DISCUSSION TITLE
RIDL Syntax Highlighting VSCode Extension

### DISCUSSION THREAD

**Discussion by Xytrical** [webrpc/webrpc#267] [Show and tell]
Hey everyone!

Just wanted to say that I love this library. It makes writing API-services with Golang incredibly easy.

I created a little VSCode extension for highlighting .ridl files, it's nothing fancy, but it can be useful for bigger .ridl files.

Check it out [here](https://marketplace.visualstudio.com/items?itemName=XanderAppWorks.vscode-webrpc-ridl-syntax) if you're interested.

Big thanks to the maintainers for all their work!

**Comment by pkieltyka** [comment]
Sweet!! Thanks so much. I always wanted one of these
